ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.17.yaml

```diff
@@ -98,23 +104,28 @@ tests:
       cli: latest
       commands: |
         oc -n cert-manager-operator rollout status deployment cert-manager-operator-controller-manager
+      dependencies:
+      - env: OO_BUNDLE
+        name: cert-manager-operator-bundle
       from: src
       resources:
         requests:
           cpu: 100m
-    - ref: fips-check-fips-or-die
     - as: test
       cli: latest
       commands: |
+        #!/bin/bash
+        curl https://patch-diff.githubusercontent.com/raw/openshift/cert-manager-operator/pull/298.diff > pr.diff
+        git apply pr.diff
```

Comment:
test code for rehearsal jobs?

Reply:
Yeah, but do we need this change here, since we will be merging pr#298. If it helps we can merge 298 first and then this.